lib/obj.js

```diff
@@ -206,7 +206,12 @@ function loadObj(objPath) {
                 var nx = parseFloat(result[1]);
                 var ny = parseFloat(result[2]);
                 var nz = parseFloat(result[3]);
-                var normal = Cartesian3.fromElements(nx, ny, nz, scratchCartesian);
+                var normal;
+                if(nx === 0 && ny === 0 && nz === 0) {
+                    normal = Cartesian3.UNIT_X;
+                } else {
+                    normal = Cartesian3.fromElements(nx, ny, nz, scratchCartesian);
+                }
```

Comment:
Also I have a Cesium fix: https://github.com/AnalyticalGraphicsInc/cesium/pull/5010 for smooth normals - may or may not go in though.